docs/_instructions/connection-settings.md

```diff
@@ -23,6 +23,10 @@ Only the `proxy` and `server` elements are looked at and only the in here define
 
 We provide a number of additional features over the maven syntax.
 
+In addition, you could also specify the maven settings inline.
+
+	-connection-settings: server\\;id="http://server"\\;username="myUser"\\;password="myPass"
```

Comment:
The backslashes should not be there.

Reply:
The backslashes are required. Otherwise, it shows error.

Reply:
I have the following: 

```
-bnd-driver                           =         nexus
nexus.settings                        =         server;id="http://server";username="myUser";password="myPass"
local.settings                        =         ~/.bnd/connection-settings.xml
-connection-settings                  =         ${if;${is;${driver};nexus};${nexus.settings};${local.settings}}
```

This results in the following error:

```
Specified -connection-settings: /Users/amitmondal/.bnd/connection-settings.xml}, but no such file or is directory
```

The problem doesn't occur if we escape the semicolons. That's why I escaped the semicolons in the PR.

Reply:
They are required by the way you use them since you are using them in an if. But if you write the statement out as in the example in the documentation, the backslashes MUST not be there.

Reply:
true. I agree.